overlore/mocks.py

```diff
@@ -0,0 +1,207 @@
+from __future__ import annotations
+
+import json
+from typing import Any
+
+from overlore.errors import ErrorCodes
+from overlore.jsonrpc.methods.generate_discussion.generate_discussion import Context as GenerateDiscussionContext
+from overlore.jsonrpc.methods.generate_discussion.generate_discussion import (
+    generate_discussion,
+)
+from overlore.jsonrpc.methods.spawn_npc.spawn_npc import Context as SpawnNpcContext
```

Comment:
Same as generate_discussion, maybe rename folder to have something clearer than spawn_npc.spawn_npc ?

Reply:
Same as above